python/taichi/ui/gui.py

```diff
@@ -13,6 +14,21 @@ class Gui:
     def __init__(self, gui) -> None:
         self.gui = gui  #reference to a PyGui
 
+    @contextmanager
+    def subwindow(self, name, x, y, width, height):
```

Comment:
maybe `sub_window` is better. FYI https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables